src/applications/check-in/components/layout/Footer.jsx

```diff
@@ -39,6 +41,15 @@ const Footer = ({ router, isPreCheckIn }) => {
     );
   };
 
+  const showTravelClaimHelp = () => {
+    if (appName === 'travelClaim') return true;
+
+    return (
```

Comment:
this return doesn't seem needed, since those error pages are not part of the travel app. The return here should just be `return false;`

Reply:
Ah ok, I copied it because I thought the error pages would be empty otherwise